src/login/components/LoginFormLedger/LoginFormLedger.js

```diff
@@ -1,34 +1,37 @@
 import React from 'react';
-import { bool, string, func, shape } from 'prop-types';
+import { string, func, shape } from 'prop-types';
 import { noop } from 'lodash';
+import { progressValues } from 'spunky';
 
 import Button from 'shared/components/Forms/Button';
 
 import styles from './LoginFormLedger.scss';
 
 const POLL_FREQUENCY = 1000;
 
+const { FAILED } = progressValues;
+
 const deviceInfoShape = shape({
   manufacturer: string.isRequired,
   product: string.isRequired
 });
 
 export default class LoginFormLedger extends React.Component {
   static propTypes = {
-    disabled: bool,
     poll: func.isRequired,
     publicKey: string,
     deviceInfo: deviceInfoShape,
     deviceError: string,
-    onLogin: func
+    onLogin: func,
+    progress: string
   };
 
   static defaultProps = {
-    disabled: false,
     publicKey: null,
     deviceInfo: null,
     deviceError: null,
-    onLogin: noop
+    onLogin: noop,
+    progress: 'INITIAL'
```

Comment:
Let's not use a string value here.  All the constants we need are defined in `progressValues`.